gemfiles/activesupport_6.0.gemfile

```diff
@@ -0,0 +1,12 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "activesupport", "6.0"
+
+group :test do
+  gem "webmock"
+  gem "simplecov", require: false
```

Comment:
Why are these needed?  I would expect the gemspec to hande this for us.

Reply:
It was in the Gemfile and appraisal copied them here automatically.

https://github.com/ManageIQ/manageiq-api-client/blob/master/Gemfile#L6-L9

I can move these into the gemspec as @kbrock suggests.

Reply:
Done, via @kbrock's suggestion